src/lib/promise-mkdirp.js

```diff
@@ -0,0 +1,27 @@
+var fs      = require('fs')
+   ,path    = require('path')
+   ,Promise = require('bluebird')
+   ,map     = require('lodash/collection/map')
+   ,cli     = process.AURELIA
+   ,mkdirp  = require('mkdirp')
+   ;
+
+
+var logger = cli.import('lib/logger');
+
+function mkdir(dirs, initialDirs){
```

Comment:
Does mkdir here only work with an array of directories?  Should it be able to handle either or implement a second called `mkdirs` instead?